support/prepReleaseCommit.ts

```diff
@@ -178,4 +172,8 @@ import yargs from "yargs";
 
     await fs.writeFile(readmePath, updatedReadmeContent);
   }
+}
+
+(async function () {
```

Comment:
I think this will run the script when you import the function from another module. Can you test?

Reply:
I created two files:

```js
// test.js
export function test(string) {
  console.log(string);
}

(function () {
  test("IIFE");
})();
```
and 
```js
// index.js
import { test } from "./test.js";
test("import");
```
I ran `node index.js` and this is what was logged in the console:
```bash
IIFE
import
```
So it doesn't seem like the IIFE runs the script on import, or else it would have been logged twice right?